packages/qulacs/tests/qulacs/circuit/test_convert_circuit.py

```diff
@@ -253,7 +269,9 @@ def test_convert_linear_mapped_parametric_circuit() -> None:
         qulacs.gate.CNOT(0, 2),
         qulacs.gate.ParametricRZ(2, 0.0),
         qulacs.gate.RX(0, -0.125),
-        qulacs.gate.ParametricPauliRotation((0, 1, 2), (1, 2, 3), 0.0),
+        qulacs.gate.ParametricPauliRotation(
+            cast_to_list((0, 1, 2)), cast_to_list((1, 2, 3)), 0.0
```

Comment:
In this case we can simply use lists.
(My past comment about favoring casting is to reduce performance overhead of object conversion when a tuple is already given)